Documentation/Usage.md

```diff
@@ -471,9 +476,9 @@ In case the current Swift version doesn't match any version-specific manifest,
 the package manager will pick the manifest with the most compatible tools
 version. For example, if there are three manifests:
 
-`Package.swift` (tools version 3.0)  
```

Comment:
These three lines are non-intentional, my formatter got trigger-happy. I think that's okay?